specification/resources/resource-manager/readme.md

```diff
@@ -26,971 +26,7 @@ These are the global settings for the Resource API.
-input-file:
-  - Microsoft.Resources/stable/2020-01-01/subscriptions.json
-```
-
-### Tag: package-subscriptions-2019-11
-
-These settings apply only when `--tag=package-subscriptions-2019-11` is specified on the command line.
-
-``` yaml $(tag) == 'package-subscriptions-2019-11'
-input-file:
-- Microsoft.Resources/stable/2019-11-01/subscriptions.json
-```
-
-### Tag: package-subscriptions-2019-06
-
-These settings apply only when `--tag=package-subscriptions-2019-06` is specified on the command line.
-
-``` yaml $(tag) == 'package-subscriptions-2019-06'
-input-file:
-- Microsoft.Resources/stable/2019-06-01/subscriptions.json
-```
-
-### Tag: package-subscriptions-2018-06
-
-These settings apply only when `--tag=package-subscriptions-2018-06` is specified on the command line.
-
-``` yaml $(tag) == 'package-subscriptions-2018-06'
-input-file:
-- Microsoft.Resources/stable/2018-06-01/subscriptions.json
-```
-
-### Tag: package-subscriptions-2016-06
-
-These settings apply only when `--tag=package-subscriptions-2016-06` is specified on the command line.
-
-``` yaml $(tag) == 'package-subscriptions-2016-06'
-input-file:
-- Microsoft.Resources/stable/2016-06-01/subscriptions.json
-```
-
-### Tag: package-subscriptions-2015-11
-
-These settings apply only when `--tag=package-subscriptions-2015-11` is specified on the command line.
-
-``` yaml $(tag) == 'package-subscriptions-2015-11'
-input-file:
-- Microsoft.Resources/stable/2015-11-01/subscriptions.json
-```
-
-### Tag: package-links-2016-09
-
-These settings apply only when `--tag=package-links-2016-09` is specified on the command line.
-
-``` yaml $(tag) == 'package-links-2016-09'
-input-file:
-- Microsoft.Resources/stable/2016-09-01/links.json
-```
-
-### Tag: package-managedapplications-2019-07
-
-These settings apply only when `--tag=package-managedapplications-2019-07` is specified on the command line.
-
-``` yaml $(tag) == 'package-managedapplications-2019-07'
-input-file:
-- Microsoft.Solutions/stable/2019-07-01/managedapplications.json
-```
-
-### Tag: package-managedapplications-2018-06
-
-These settings apply only when `--tag=package-managedapplications-2018-06` is specified on the command line.
-
-``` yaml $(tag) == 'package-managedapplications-2018-06'
-input-file:
-- Microsoft.Solutions/stable/2018-06-01/managedapplications.json
-```
-
-### Tag: package-managedapplications-2017-09
-
-These settings apply only when `--tag=package-managedapplications-2017-09` is specified on the command line.
-
-``` yaml $(tag) == 'package-managedapplications-2017-09'
-input-file:
-- Microsoft.Solutions/stable/2017-09-01/managedapplications.json
-```
-
-### Tag: package-managedapplications-2016-09
-
-These settings apply only when `--tag=package-managedapplications-2016-09` is specified on the command line.
-
-``` yaml $(tag) == 'package-managedapplications-2016-09'
-input-file:
-- Microsoft.Solutions/preview/2016-09-01-preview/managedapplications.json
+tag: profile-hybrid-2019-03-01
```

Comment:
```suggestion
```
there's no default tag in this case, we keep this tag just for record, it's not the default tag.